AutomatedReview/JenkinsDeploymentRunbook.md

```diff
@@ -0,0 +1,87 @@
+# Jenkins Deployment Runbook
+The purpose of this runbook is to instruct Build SIG maintainers how to trigger a deployment of the O3DE Jenkins server. Deployments are automated by the pipeline detailed here: [O3DE Jenkins Pipeline Repo RFC](https://github.com/o3de/sig-build/blob/main/rfcs/rfc-bld-20220209-1-jenkins-pipeline-repo.md)
+
+## Getting Started 
+
+The Build SIG maintains the Jenkins pipeline and approves changes to the infrastructure required to run it. A select group of maintainers within the SIG have access to the AWS account hosting the underlying infrastructure. Contact the chair/co-chair of [#sig-build](https://discord.com/channels/805939474655346758/816043576034328636) for more info. 
+
+### Resources
+
+| Jenkins Instance | Description | Git Repo/Branch |
+| --- | --- | --- |
+| https://jenkins.build.o3de.org/ | O3DE Prod Instance | o3de-jenkins-pipeline / branch: o3de-prod |
+| https://jenkins-sandbox.build.o3de.org/ | O3DE Sandbox Instance | o3de-jenkins-pipeline / branch: o3de-sandbox
```

Comment:
Is that going to have the same attributes as the prod version or a reduced set of resources?

Reply:
It will have a reduced set like fewer builds nodes, platforms, etc. For the stack, I have tasks to provide arguments to the CDK package adjust the resources allocated to the pipeline.